java/src/main/java/com/cybersource/ws/client/HttpClientConnection.java

```diff
@@ -231,6 +237,28 @@ public boolean retryMethod(
             return false;
         }
     }
+    
+	@Override
+	void logRequestHeaders() {
+		Header[] headers=postMethod.getRequestHeaders();
```

Comment:
there is no need to convert to map. can we directly print headers ?

Reply:
headers array has key and value pairs.If im printing just headers, then the object is getting printed. Removed the map and printed as ArrayList.